update-dois/src/main/java/org/reactome/release/updateDOIs/UpdateDOIs.java

```diff
@@ -0,0 +1,40 @@
+package org.reactome.release.updateDOIs;
+
+import java.io.IOException;
+import java.util.Properties;
+
+import org.gk.model.GKInstance;
+import org.gk.persistence.MySQLAdaptor;
+import org.gk.model.PersistenceAdaptor;
+
+public class UpdateDOIs {
+
+  public static void main( String[] args ) {
+
+    MySQLAdaptor testReactomeDBA = null;
+    MySQLAdaptor gkCentralDBA = null;
+    try {
+      String user = "root";
+      String password = "root";
+      String host = "localhost";
+      String databaseTest = "test_reactome_64";
+      String databaseGk = "gk_central";
+      int port = 3306;
+      // Set up db connections.
+      testReactomeDBA = new MySQLAdaptor(host, databaseTest, user, password, port);
```

Comment:
All of the values passed into the MySQLAdaptor constructor should come from a properties file. Look to release-qa for some simple examples of how this can be done.